classes/Cart.php

```diff
@@ -4178,6 +4178,18 @@ public function deleteCustomizationToProduct($id_product, $index)
             AND `index` = ' . (int) $index
         );
 
+        $hasRemainingCustomData = Db::getInstance()->getValue(
+            'SELECT 1 FROM `' . _DB_PREFIX_ . 'customized_data`
+            WHERE `id_customization` = ' . (int) $cust_data['id_customization']
+        );
+
+        if (!$hasRemainingCustomData) {
```

Comment:
Is it really needed to check for existence before removing rows?

Reply:
One customization can have several customized_data (we can put several custom fields on a product). So if only one custom field is removed, we have to keep the customization entry

Reply:
Oh yes, missreading the `!` :sweat_smile: